src/html/options.html

```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
 <html>
-	<head> 
-		<meta charset="UTF-8">
-		<title><%= htmlWebpackPlugin.options.title %></title>
-	</head>
-	<body>
-		<div id="options-root"></div>
-	</body>
+  <head>
```

Comment:
this still seems to be using 2 spaces somehow?

Reply:
missed adding .html to the script